css-brochure-form/styles.css

```diff
@@ -0,0 +1,83 @@
+h1 {
+  color: #e64219;
+  text-align: center;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 25px;
+  position: relative;
+  top: 25px;
+}
+
+label {
+  color: black;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  display: block;
+  padding-top: 20px;
+  font-size: 14px;
+}
+
+input {
+  display: block;
+  margin-right: auto;
+  margin-left: auto;
+  border-radius: 5px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  text-align: left;
+  padding-left: 10px;
+  border-color: lightgrey;
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  width: 500px;
+  border: 1px solid grey;
+}
+
+#submit {
+  top: 10px;
+  position: relative;
+  left: 190px;
+  display: block;
+  text-align: center;
+  color: white;
+  background-color: #e64219;
+  border-radius: 20px;
+  width: 120px;
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 17px;
+  border: none;
+}
+
+#company {
+  padding-left: 530px;
+}
+
+#nombre {
+  padding-left: 530px;
+}
+
+#hiring {
+  padding-left: 530px;
+}
```

Comment:
This is waaaaay too much padding. Use a different method to position these where they need to go.